tailwind-annotator.config.mjs

```diff
@@ -9,4 +9,14 @@ export default {
     // This module references `sidebar-frame` and related classes
     './src/annotator/sidebar.{js,ts,tsx}',
   ],
+  theme: {
+    extend: {
+      borderRadius: {
+        // Equivalent to tailwind defaults, but overriding values from frontend-shared preset
+        // Once the preset stops defining borderRadius, we can remove this
+        DEFAULT: '0.25rem',
+        lg: '0.5rem',
+      },
+    }
+  }
```

Comment:
This makes annotator components be always rounded.

This was their behavior already, but it was set via hardcoded border radius `rounded-[4px]`.

Those classes have been replaced by `rounded`, and this config ensures they keep looking the same regardless the feature being enabled or not, while it becomes ready for the future, when we just use tailwind defaults.